app/services/notifications/moderation.rb

```diff
@@ -5,6 +5,7 @@ module Moderation
 
     def self.available_moderators
       User.with_role(:trusted).joins(:notification_setting)
+        .where("last_reacted_at > ?", 1.week.ago)
```

Comment:
nit (non-blocking): I personally like this a bit better
```suggestion
        .where(last_reacted_at: 1.week.ago..)
```
which would produce this sql query. keep in mind that it's using ">=" here.
```
User Load (0.8ms)  SELECT "users".* FROM "users" WHERE "users"."last_reacted_at" >= $1  [["last_reacted_at", "2023-01-11 18:59:05.198243"]]
```